app/client/src/pages/aboutus.tsx

```diff
@@ -1,52 +1,111 @@
 import * as React from 'react';
-import { Paper, Grid, Typography } from '@material-ui/core';
+import Image from 'next/image';
+import { Grid, Typography, Hidden } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme) => ({
     root: {
         width: '100%',
-        height: '100%',
+        minHeight: '85vh',
+        paddingLeft: '1rem',
     },
-    paper: {
-        padding: theme.spacing(3),
+    section: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        gap: '1rem',
+        textAlign: 'center',
+    },
+    paragraph: {
+        fontSize: 'large'
     },
 }));
 
-const AboutUs = () => {
+export default function AboutUs() {
     const classes = useStyles();
+
     return (
-        <Grid className={classes.root} container direction='column' justify='center' alignItems='center' spacing={3}>
-            <Grid item>
-                <Typography variant='h1'>About Us</Typography>
+        <Grid container alignItems='center' className={classes.root} justify='center' spacing={2}>
+            
+            <Grid item xs={12} className={classes.section}>
+                <Grid item xs={4} sm={3} md={2}>
+                    <Image
+                        src='https://i.ibb.co/qmXt8Fw/prytaneum-logo-filled.png' 
+                        width={1107}
+                        height={1108}
+                        objectFit='contain'
+                    />
+                </Grid>
+                <Typography variant='body1' className={classes.paragraph}>
+                    Technology to Enhance Democracy
+                </Typography>
+                <Typography variant='h4'>
+                    Our Journey
+                </Typography>
             </Grid>
-            <Grid item>
-                <Typography>
-                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse ultricies pulvinar eros, in
-                    tincidunt arcu rutrum id. Pellentesque ac elit at felis efficitur ullamcorper pretium vel est.
-                    Mauris ac velit hendrerit, rutrum lectus eget, aliquet metus. Donec venenatis dolor quis arcu
-                    fermentum blandit. Maecenas finibus ac nibh et ultrices. Praesent non mattis odio, ac cursus turpis.
-                    Cras euismod quam eu leo euismod, id semper felis aliquam. Fusce vulputate pretium dui id faucibus.
-                    Mauris tristique nibh ut leo rhoncus sodales. Nam posuere lectus quis augue volutpat, at interdum
-                    orci malesuada. Aenean cursus nisi eu pharetra condimentum. Phasellus vitae urna et ex gravida
-                    eleifend.
+            <Grid item xs={12} sm={4} md={3} className={classes.section}>
+                <Image
+                    src='https://i.ibb.co/4YYwHx8/directors.png' 
```

Comment:
We should try building this image directly into our assets rather than externally linking it.